### PR TITLE
Implement Viewport Paint

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Appears to work all fine.

### CODE REVIEW

**Review by AaronVanGeffen — APPROVED** [review]
All looks to be working correctly :+1:

File: src/OpenLoco/Paint/Paint.cpp [hunk 1]
```diff
@@ -764,6 +764,12 @@ namespace OpenLoco::Paint
         call(0x0045EA23);
     }
 
+    // 0x0045A60E
+    void PaintSession::drawStringStructs()
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Paint/Paint.cpp:768]: What does this function do?

File: src/OpenLoco/Viewport.cpp [hunk 2]
```diff
@@ -35,17 +46,282 @@ namespace OpenLoco::Ui
...
+            if (station.owner == CompanyId::null)
+            {
+                return Colour::grey;
+            }
+            else
+            {
+                return CompanyManager::getCompanyColour(station.owner);
+            }
+        }(),
+                                                    isHovered ? 0 : 1);
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Viewport.cpp:105]: What's happening here, formatting-wise? :joy:

File: src/OpenLoco/Paint/Paint.cpp (hunk 1 above)
Comment (duncanspumpkin) [reply src/OpenLoco/Paint/Paint.cpp:768]: Draws the string structs :p. String structs are basically just used for the floating currency entities.

File: src/OpenLoco/Viewport.cpp (hunk 2 above)
Comment (duncanspumpkin) [reply src/OpenLoco/Viewport.cpp:105]: Yeah it all went a bit odd after i joined the two functions (hovered and not hovered). I'll split up the lambda so it's easier to read.

File: src/OpenLoco/Paint/Paint.cpp (hunk 1 above)
Comment (AaronVanGeffen) [reply src/OpenLoco/Paint/Paint.cpp:768]: Ah, that wasn't obvious to me. I'm sure it'll make sense once implemented, though.

**Pull request merged**